dist/index.html

```diff
@@ -2,11 +2,96 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Activity Tracker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>🔥 Fitlit</title>
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+   <section class="landing-page">
+    <section class="logo">
+      <section>
+        <img>
+        <h2>fitlit</h2>
+      </section>
+      <section class="dashboard">
+        <img>
+        <h1>Welcome, Name!</h1>
+        <h2> 7.11.2021</h2>
+        <div>
+          <button class="hydration"></button>
+          <button class="sleep"></button>
+          <button class="activity"></button>
+        </div>
+        <p>Step Goal Comparison</p>
+        <button class="user-info"></button>
+      </section>
+    </section>
+   </section>
+
+   <section class="hydration hidden">
+    <section class="logo">
+      <img>
+      <h2>fitlit</h2>
+    </section>
+    <section class="data-display">
+      <h1>Hydration</h1>
+      <img>
+      <div class="bar-graph"></div>
+    </section>
+   </section>
+
+   <section class="sleep hidden">
+    <section class="logo">
+      <img>
+      <h2>fitlit</h2>
+    </section>
+    <section class="data-display">
+      <h1>Sleep</h1>
+      <img>
+      <div class="bar-graph"></div>
+      <h3>All Time Averages</h3>
+      <div>
+        <p>Sleep Quality: </p>
+        <p>Sleep Hours: </p>
+      </div>
+    </section>
+   </section>
+
+   <section class="activity hidden">
+    <section class="logo">
+      <img>
+      <h2>fitlit</h2>
+    </section>
+    <section class="data-display">
+      <h1>Activity</h1>
+      <img>
+      <div class="bar-graph"></div>
+      <div class="activity-buttons">
+        <button>Steps</button>
```

Comment:
Finish researching the functionality of what will happen on button click - how might we refactor HTML to better showcase what buttons lead to?